packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -63,7 +67,11 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
       const pending = zone.mapStore('pending', {
         durable: true,
       });
-      return { operators, pending };
+      /** @type {MapStore<string, MapStore<string, RiskAssessment>>} */
+      const risks = zone.mapStore('risks', {
+        durable: true,
+      });
```

Comment:
Whether a store is durable is supposed to be a function of what sort of zone, I think. I think `durable: true` is a noop for a durable zone and ignored by a heap zone.

```suggestion
      const risks = zone.mapStore('risks');
```

Reply:
Okay, removed here and above for the other stores